google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -94,16 +102,22 @@ def __init__(self, *args: Any) -> None:
 class InstanceMetadata:
     ip_addrs: Dict[str, Any]
     context: ssl.SSLContext
+    expiration: datetime.datetime
+    seconds_until_refresh: int
```

Comment:
Hmm, why are we storing this value on the object? Won't it a) become invalid almost instantly and b) only be used once to schedule the next refresh?

Reply:
You're right, it doesn't really make sense. I think I just wanted to do all the calculations while creating the ephemeral cert. Updated to only store the expiration